PopPUNK/refine.py

```diff
@@ -67,6 +67,13 @@ def refineFit(distMat, sample_names, start_s, mean0, mean1,
     sys.stderr.write("Decision boundary starts at (" + "{:.2f}".format(start_point[0])
                       + "," + "{:.2f}".format(start_point[1]) + ")\n")
 
+    # calculate distance between start point and means if none is supplied
+    if min_move is None:
+        min_move = ((mean0[0] - start_point[0])**2 + (mean0[1] - start_point[1])**2)**0.5
+    if max_move is None:
+        max_move = ((mean1[0] - start_point[0])**2 + (mean1[1] - start_point[1])**2)**0.5
+    print('Start: ' + str(start_point) + '\nMean0: ' + str(mean0) + '\nMean1: ' + str(mean1) + "\nMax move is " + str(max_move) + '\nMin move is: ' + str(min_move))
```

Comment:
```suggestion
```

Reply:
Thanks, meant to remove this debug message but forgot.